db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijImpl.java

```diff
@@ -1,7 +1,11 @@
 package com.splicemachine.db.impl.tools.ij;
 
+import com.splicemachine.db.iapi.tools.i18n.LocalizedOutput;
 import com.splicemachine.db.iapi.tools.i18n.LocalizedResource;
 import com.splicemachine.db.tools.JDBCDisplayUtil;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import sun.misc.Signal;
```

Comment:
Using `sun.misc.Signal` is a bit "controversial" to say the least (the API contains a lot of non-standard OS-specific code and it was supposed to be removed but somehow survived as [listed here](https://stackoverflow.com/a/34188676)).

Maybe that's OK considering the benefit this PR brings, I am guessing you tested it already on Mac/Linux and maybe Windows? if not, then could you please test the client there, there is a [guide](https://splicemachine.atlassian.net/wiki/spaces/PD/pages/976289824/How+to+use+Windows+SQL+Shell) on how to do so.

Reply:
i didn't tested it on linux and windows, but now i did. Works on both Windows and Linux. What can happen is that sun.misc.Signal is not allowed to be executed, that's why i added a try/catch block around this.
However, it seems like on some systems there is no sun.misc.Signal so i will change this by using reflection.

Reply:
done